implementations/rust/ockam/ockam_node/src/node.rs

```diff
@@ -31,16 +32,57 @@ pub fn start_node() -> (Context, Executor) {
 }
 
 /// Utility to setup tracing-subscriber from the environment
+#[cfg(not(feature = "console"))]
```

Comment:
I think we don't want to duplicate this entirely -- we should instead move the CFG into the body of `setup_tracing`. Otherwise I think these will drift.